doc/rtd/topics/faq.rst

```diff
@@ -3,19 +3,191 @@
 FAQ
 ***
 
-Getting help
-============
+How do I get help?
+==================
 
 Having trouble? We would like to help!
 
+- First go through this page with common questions
 - Use the search bar at the upper left to search these docs
 - Ask a question in the ``#cloud-init`` IRC channel on Freenode
 - Join and ask questions on the `cloud-init mailing list <https://launchpad.net/~cloud-init>`_
-- Find a bug? `Report bugs on Launchpad <https://bugs.launchpad.net/cloud-init/+filebug>`_
+- Find a bug? Check out the :ref:`reporting_bugs` topic for
+  how to report one
 
+Where are the logs?
+===================
 
-Media
-=====
+Cloud-init uses two files to log to:
+
+- `/var/log/cloud-init-output.log`: captures the output from each stage of
+  cloud-init when it runs
+- `/var/log/cloud-init.log`: very detailed log with debugging output,
+  detailing each action taken
+
+Be aware that each time a system boots, new logs are appended to these files.
+Therefore, the files may have more than one boot worth of information present.
+
+When reviewing these logs look for any errors or Python tracebacks to check
+for any errors.
+
+Where are the configruation files?
+==================================
+
+Inside the `/var/lib/cloud/` directory there are two important subdirectories:
+
+instance
+--------
+
+The `/var/lib/cloud/instance` directory contains the information cloud-init
```

Comment:
symbolic link points to the most recently used instance-id directory and contains....

Reply:
fixed